include/wx/qt/calctrl.h

```diff
@@ -78,6 +79,7 @@ class WXDLLIMPEXP_ADV wxCalendarCtrl : public wxCalendarCtrlBase
 private:
     void Init();
     void UpdateStyle();
+    bool IsQDateValid(const QDate &date) const;
```

Comment:
I think this function is an implementation detail, so I'd keep it in `src/qt/calctrl.cpp` instead of making it a member -- it doesn't really need to be a member function, and so shouldn't be one.